app/views/posts/index.html.erb

```diff
@@ -1 +1,55 @@
-<h1> show a post</h1>
\ No newline at end of file
+<h1 class ="usersTitle" > All the post of the user </h1>
+<div class= "userProfile">
+  <div class= "usersContainer">
+    <div class= "usersPhotoContainer">
+        <%= image_tag @user.photo , class: "usersPhoto" %>
+        </div>
+        <div class="usersNameAndPost">
+        <p class = "usersName">
+            <%= @user.name %>
+        </p>
+        <p class = "usersPost" >
+            Number of posts: <%= @user.posts_counter %>
+        </p>
+    </div>
+  </div>
+    <div class = "postsContainer">
+    <% @posts.each do |post|%>
+      <div class = "postsAndComments">
+        <div class= "post">
+          <h2>
+            <%= link_to post.title, user_post_path(@user.id, post.id)  %>
+          </h2>
+          <p>
+          <%= post.text.length > 100 ? post.text[0..100].concat('...') : post.text  %>
+          </p>
+          <div class = "counters">
+            <span> 
+              Comments: <%= post.comments_counter == nil ?  0 : post.comments_counter  %>
+            </span>
+            ,
+            <span> 
+              Likes: <%= post.likes_counter == nil ? 0 : post.likes_counter  %>
+            </span>
+          </div>
+        </div>
+        <div class= "postComments">
+        <% if Comment.where(post_id_id: post.id).any? %>
+          <% post.recent_comments.each do |comment| %>
+            <p>
+              <%= User.find(comment.author_id_id).name %>
```

Comment:
**Your app is working as expecte, all of them are normal. ** 

[OPTIONAL] However, I strongly suggest you to replace `author_id_id`, and `post_id_id` to `author_id`, and `post_id`. By this solution you could save some space and improve readability of your code!